### DISCUSSION TITLE
custom made gpx appears twice in other programs

### DISCUSSION THREAD

**Discussion by Martinelli1978** [osmandapp/OsmAnd#18023] [Maps Related]
I like to create my own gpx tracks based on walks I find online. No problems there. I create them, use them, other people can use them in osmand without problems etc. But! Everytime I or someone else wants to use these custom made gpx in another program like Komoot, dynamic watch, gpx viewer, the gpx appears twice. Once based on the points I created on the map (straight line) and once based on the fact that I told osmand to follow the walking paths between those points. If I send these tracks to other people and they open them in osmand the track only appears once. This is really annoying. I have plans for a website where I deliver my gpx routes for free but the gpx should be able to be used in multiple programs. How can I stop getting this duplicate? Example attached. I would have liked to attach the gpx too, but I don't see how.
<table style="width:100%">
  <tr>
<th><img src="https://github.com/osmandapp/OsmAnd/assets/90757989/b08019fa-3465-41f0-a15a-99cc65afcf4a"  width="300" height="auto"/> </td>
  </tr>
</table>

**Comment by DmitryAlexei** [comment]
@Martinelli1978
 You can try asking the support team of other apps why they are not correctly displaying tracks imported from OsmAnd.
 This is not OsmAnd issue, as the tracks are displaying correctly.

**Comment by Martinelli1978** [comment]
Thank you. I actually thought it would be an issue for osmand. Since the display of the tracks is wrong in multiple programs, not just one, isn't this something in the way osmand builds or saves a track?
It will probably be the end of the project I was planning, because this would force other people to use osmand when they are using one of my tracks and I want them to decide for themselves if they want to use their garmin, komoot or whatever. I guess I will try another program myself to create the tracks from now on to see if that makes a difference.

**Comment by DmitryAlexei** [comment]
Every app uses global GPX standard with its own particularities. 
For OsmAnd you can see https://osmand.net/docs/technical/osmand-file-formats/osmand-gpx

As a workaround, you can try to use *Simplified track* while saving your tracks in OsmAnd
https://osmand.net/docs/user/plan-route/create-route#save-as-new-track

**Comment by Martinelli1978** [comment]
Thank you, I will try that and will read some more about the technical stuff regarding gpx!

**Reply by Martinelli1978** [reply]
Saving the track as simplified track seems to work, thanks again!
